content/en/guides/create-program.md

```diff
@@ -222,11 +222,11 @@ For example, your policy could require engineers to receive approval from your o
 
 Also needed are policies to formalize and establish legal compliance procedures and to assure executive oversight for the program. (See our ebook, [Open Source Compliance in the Enterprise](https://www.linuxfoundation.org/compliance-and-security/2018/12/open-source-compliance-in-the-enterprise/), for a detailed compliance program.)
 
-You’ll also want to map out how you’ll handle the software tools that will make much of the compliance and code testing work possible by automating it and streamlining procedures for developers and contributors. For example, the [I think we need to mention SPDX and OpenChain tools from The Linux Foundation, along with help from your and working with internal supply chain teams, will internally to ensure that code brought in by third parties is also compliant.]
+You’ll also want to map out how you’ll handle the software tools that will make much of the compliance and code testing work possible by automating it and streamlining procedures for developers and contributors. We'd like to mention  along with help from internal supply chain teams (should you have them) to ensure that code brought in by third parties is also compliant.
```

Comment:
The sentence starting at "We'd like to mention…" is an incomplete thought.

Also, you don't "like to mention." Do it or don't do it. There is no try/like. ;-)

Reply:
Vestigal leftovers from moving things around. Good catch :fishing_pole_and_fish: